extensions/github/package.json

```diff
@@ -408,6 +408,7 @@
     "@graphql-codegen/typescript": "^4.0.7",
     "@graphql-codegen/typescript-graphql-request": "^6.2.0",
     "@graphql-codegen/typescript-operations": "^4.2.1",
+    "@parcel/watcher": "^2.4.1",
```

Comment:
Is that needed?

Reply:
I was seeing some warnings during development mode. The GraphQL generator script required this dependency, so I added it to devDependency just in case.